adal/src/main/java/com/microsoft/aad/adal/TokenCacheItem.java

```diff
@@ -106,6 +111,9 @@ private TokenCacheItem(final String authority, final AuthenticationResult authen
         mRawIdToken = authenticationResult.getIdToken();
         mRefreshtoken = authenticationResult.getRefreshToken();
         mFamilyClientId = authenticationResult.getFamilyClientId();
+        if (authenticationResult.getExtendedExpiresOn() != null) {
```

Comment:
yup

---
In reply to: [71007654](https://github.com/AzureAD/azure-activedirectory-library-for-android/pull/676#discussion_r71007654) [](ancestors = 71007654)